src/plugins/tts/FestivalTTSEngine.js

```diff
@@ -91,10 +91,9 @@ export default class FestivalTTSEngine extends AbstractTTSEngine {
    * See https://stackoverflow.com/questions/12206631/html5-audio-cant-play-through-javascript-unless-triggered-manually-once
    * @return {PromiseLike}
    */
-  iOSCaptureUserIntentHack() {
+  async iOSCaptureUserIntentHack() {
     const sound = soundManager.createSound({ url: SILENCE_1MS[this.audioFormat] });
-    return new Promise(res => sound.play({onfinish: res}))
-      .then(() => sound.destruct());
+    await sound.play({onfinish: await sound.destruct()});
```

Comment:
The order we want this to run is:

1) createSound
2) Play sound;
3) Once sound is finished playing (! this is the async thing), destruct sound.

So we'll still need the `await new Promise(res => sound.play({onfinish: res}))`; That converts the callback `onfinish` to a promise we can `await` !